foodnet/__init__.py

```diff
@@ -1 +1 @@
-
+__author__ = 'valberg'
```

Comment:
Is this really necessary? How about we have a separate file called contributors.rst for this sort of stuff?